agent/app/agent_unix.go

```diff
@@ -48,7 +48,7 @@ const initPID = 1
 var awsVPCCNIPlugins = []string{ecscni.ECSENIPluginName,
 	ecscni.ECSBridgePluginName,
 	ecscni.ECSIPAMPluginName,
-	ecscni.ECSAppMeshPluginName,
+	//ecscni.ECSAppMeshPluginName,
```

Comment:
why is it commented out?

Reply:
Appmesh is not yet supported with how the CNI plugins were installed. This will be added to future work for the project.